pkg/slayers/path/hopfield.go

```diff
@@ -79,40 +79,60 @@ type HopField struct {
 // @ preserves acc(slices.AbsSlice_Bytes(raw, 0, HopLen), R45)
 // @ ensures   h.Mem()
 // @ ensures   err == nil
+// @ ensures   h.CorrectlyDecodedHF(raw, 0, 0, HopLen)
 // @ decreases
 func (h *HopField) DecodeFromBytes(raw []byte) (err error) {
 	if len(raw) < HopLen {
 		return serrors.New("HopField raw too short", "expected", HopLen, "actual", len(raw))
 	}
-	//@ preserves acc(h)
-	//@ preserves acc(slices.AbsSlice_Bytes(raw, 0, HopLen), R46)
-	//@ ensures h.ConsIngress >= 0
-	//@ ensures h.ConsEgress >= 0
-	//@ decreases
-	//@ outline(
-	//@ unfold acc(slices.AbsSlice_Bytes(raw, 0, HopLen), R46)
+	// @ requires  len(raw) >= HopLen
+	// @ preserves acc(h)
+	// @ preserves acc(slices.AbsSlice_Bytes(raw, 0, HopLen), R46)
+	// @ ensures   h.ConsIngress >= 0
+	// @ ensures   h.ConsEgress >= 0
+	// @ ensures   let h1 := BytesToIO_HF(raw, 0, 0, HopLen) in
+	// @           let h2 := h.ToIO_HF() in
+	// @           h1.InIF2 == h2.InIF2 &&
+	// @           h1.EgIF2 == h2.EgIF2
+	// @ decreases
+	// @ outline(
+	// @ unfold acc(slices.AbsSlice_Bytes(raw, 0, HopLen), R46)
```

Comment:
Considering all the effort that went into making Gobra/silicon faster in the last few months, I think that these outline blocks are now unnecessary because
1) this package is not problematic in the first place
2) we can afford it now, even if this is ever so slightly slower (which probably is not the case anymore), and the annotation overhead is not worth it